opensrp-app/res/layout/smart_register_pagination.xml

```diff
@@ -1,39 +1,43 @@
-<?xml version="1.0" encoding="utf-8"?>
-
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Not sure why we have this large diff here?

Reply:
Actually, it looks like a large change but it's the minimal change you can check the diff

Reply:
If it's a formatting issue, please revert to the previous format.

Reply:
locally not showing any formatting issue